common/changes/@itwin/core-backend/fontMapCase_2024-11-14-21-53.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "IModel.test.ts now validates that FontMap.getFont uses case insensitive matching for font names",
```

```suggestion
      "comment": "",
```

change logs are for end consumers, and they wont care about tests like this

---

Thanks, suggestion committed